MarketplaceService/Startup.cs

```diff
@@ -25,27 +22,32 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
```

Comment:
Er moet nog authentication toegevoegd worden (JWT) hier en in de controllers.

Reply:
Er is nog geen authenticatie methode voor derde services in de account service dus die staat nog on hold